recipe/meta.yaml

```
@@ -28,6 +24,7 @@ requirements:
   run:
     - python
     - python-dateutil
+    - pytz
```

```suggestion
    - pytz >2021.1
```
https://github.com/kiorky/croniter/blob/3.0.3/requirements/base.txt#L2
The constraint is important too.